tests/tinderbox_scraper/test_tinderbox_scraper.py

```diff
@@ -7,12 +7,12 @@
 import os
 import urllib
 
+import pytest
+
 from mozdownload import TinderboxScraper
 from mozdownload.utils import urljoin
 
-import mozhttpd_base_test as mhttpd
-
-
+'''
 firefox_tests = [
```

Comment:
Please remove all the old configs. The list of parametrized entries I will check then. Today I won't have the time for it anymore.